lib/fluent/supervisor.rb

```diff
@@ -82,6 +82,16 @@ def run_rpc_server
         end
         nil
       }
+      @rpc_server.mount_proc('/api/plugins.flushBuffersAndKillWorkers') { |req, res|
```

Comment:
killWorkers operation is not related with plugins so adding it to `plugins` prefix seems not good.
processes is better.